src/components/ParticipantInfo/AudioTrackPublicationInfo/AudioTrackPublicationInfo.tsx

```diff
@@ -37,7 +39,9 @@ export const AudioTrackInfo: React.FC<{
 
   return (
     <>
-      <Datum label="isEnabled" value={isEnabled} />
+      {/* Only show isSwitchedOff and trackSwitchOffReason for remote tracks */}
+      {isSwitchedOff !== undefined && <Datum label="isSwitchedOff" value={isSwitchedOff} />}
+      {trackSwitchOffReason !== undefined && <Datum label="Switch Off Reason" value={trackSwitchOffReason} />}
       <Datum label="Bandwidth" value={trackBandwidth?.toLocaleString() + 'kbps'} />
```

Comment:
Noticed that when `trackBandwidth` is undefined it looks like this:

![Screen Shot 2022-08-11 at 4 22 21 PM](https://user-images.githubusercontent.com/77076398/184258569-bf182ae5-7ecc-4084-9190-ac2e92bddcf3.png)

Maybe if it's undefined we removed the `kbps`.. or hide it?